packages/x-data-grid/src/hooks/features/editing/useGridCellEditing.ts

```diff
@@ -234,6 +234,19 @@ export const useGridCellEditing = (
     [apiRef],
   );
 
+  const runIfNoFieldErrors =
+    <Args extends any[]>(callback?: (...args: Args) => void) =>
```

Comment:
Nitpick: better type safety
```suggestion
    <Args extends Parameters<GridEventListener<'cellEditStop'>>>(callback?: (...args: Args) => void) =>
```